### DISCUSSION TITLE
No Logs in bro-* dashboards

### DISCUSSION THREAD

**Discussion by vijaytejak** [Security-Onion-Solutions/securityonion#5181] [Unsupported Versions] [answered]
Hi, I have set up Security Onion and running for a month, I see these bro dashboards, but I don't see any logs in it, I don't see the logs in any of the Bro-Hunting categories in Kibana.

what data is required to display the logs in these dashboards, I also have RITA are these RITA logs?

Need help.

![image](https://user-images.githubusercontent.com/14872281/129858109-795f8850-4bb8-4ffe-a183-1e5c6bb7b80b.png)

**Comment by gcsrellis** [comment]
Have you verified you are ingesting traffic from your sensor node?  A quick way to tell is to ifconfig and look at the RX under your span interface.  Should have tons of data coming in and essentially none under TX.  From there you can troubleshoot the rest.

**Comment by dougburks** [comment, marked as answer]
The screenshot is showing the old 16.04 dashboards. Please make sure you are looking at the new `Security Onion` dashboards. From https://docs.securityonion.net/en/2.3/kibana.html#dashboards:
> We’ve included the old 16.04 dashboards in case you performed an in-place upgrade and have any old 16.04 data. These dashboards are named with the z16.04 prefix and will only show old 16.04 data. The new Security Onion 2 dashboards are all named with the Security Onion prefix and they should be used for any new data going forward.

**Comment by vijaytejak** [comment]
yeah, I got it now, those dashboards are not required, I am using 2.3.50
thanks for the response.
